src/libgrate/grate-3d.c

```diff
@@ -659,11 +659,13 @@ static void grate_3d_set_texture_desc(struct host1x_pushbuf *pb,
 				      struct host1x_pixelbuffer *pixbuf,
 				      unsigned max_lod,
 				      unsigned wrap_mode,
-				      bool mip_filter,
-				      bool mag_filter,
-				      bool min_filter)
+				      bool mipmap_enabled,
+				      bool min_filter_linear_within,
+				      bool min_filter_linear_between,
+				      bool mag_filter_linear)
```

Comment:
The "min"-filter is what happens to a single texture image when it's being minified (similar to magfilter), and the "mip"-filter is how the results of the min/mag-filters gets combined. So unless I'm confused, I think that would be:

MINFILTER == MINFILTER_LINEAR_WITHIN
MIPFILTER == MINFILTER_LINEAR_BETWEEN